deps/wazuh_testing/wazuh_testing/tools/performance/statistic.py

```diff
@@ -1,25 +1,36 @@
+
+
+
 # Copyright (C) 2015-2021, Wazuh Inc.
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 import csv
 import json
 import logging
+import requests
+import urllib3
+import wazuh_testing.tools as tls
 from datetime import datetime
 from os.path import basename, isfile, join, splitext
 from re import sub
 from tempfile import gettempdir
 from threading import Thread, Event
-from time import sleep
+from time import sleep, time
+from wazuh_testing.tools.performance import statistic_headers as headers
 
-import wazuh_testing.tools as tls
+urllib3.disable_warnings()
+
+API_URL="https://localhost:55000"
```

Comment:
Hardcoded api url and port

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4970/commits/8128e8a9a2702d28ba4216603d8539104506be48